curator-framework/src/main/java/org/apache/curator/framework/CuratorFrameworkFactory.java

```diff
@@ -380,16 +383,6 @@ public String getNamespace()
             return namespace;
         }
 
-        public String getAuthScheme()
```

Comment:
I think these methods should stay and be marked as deprecated so this is not a breaking change. The old methods can either return return the first auth or throw an error if there is more than 1 auth.